### PR TITLE
Feature/pauda

### PR DESCRIPTION
**Pull request by aymanm**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

Adding a recipe for Pauda.
See link for more info https://ab.inf.uni-tuebingen.de/software/pauda

### CODE REVIEW

File: recipes/pauda-mergeblast/merge.blast.pl [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env perl
+# xiechaos@gmail.com
```
Comment (bgruening) [inline recipes/pauda-mergeblast/merge.blast.pl]: This script is not needed in this PR, isn't it?

Comment (aymanm) [inline recipes/pauda-mergeblast/merge.blast.pl]: Correct, its not needed. It is a complementary script for merging BLAST files produced by PAUDA.

Comment (jerowe) [inline recipes/pauda-mergeblast/merge.blast.pl]: @bgruening , I would make the case for pauda-mergeblast being in with the main pauda. They live on separate urls, but are part of the same tool.

Comment (bgruening) [inline recipes/pauda-mergeblast/merge.blast.pl]: What I mean is you are downloading this script from http://ab.inf.uni-tuebingen.de/data/software/pauda/download/merge.blast.pl ... why is this included in the PR as single file and you copy it not to PREFIX/bin, isn't it?

Comment (aymanm) [inline recipes/pauda-mergeblast/merge.blast.pl]: Sorry I am new to this. Can you please clarify?
Do you mean download the script to recipes/pauda folder and just copy it to PREFIX/bin ?

**Comment by aymanm** [comment]
@bgruening is this ok to merge?

**Comment by aymanm** [comment]
@bioconda/core any other comments on this recipe would be really appreciated

**Comment by bgruening** [comment]
Looks good now! Thanks.
General question is there any way to compile it and not download the pre-compiled stuff?

**Comment by aymanm** [comment]
I believe the compiled version is the only one available by the authors. After a quick search, I could not find the source code of the java files.

**Comment by bgruening** [comment]
Thanks @aymanm!
